includes/rest-api/class-wp-job-manager-data-stores-status.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Declaration of our Status Data Store
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Data_Stores_Status
+ */
+class WP_Job_Manager_Data_Stores_Status extends WPJM_REST_Data_Store_Abstract
```

Comment:
One thing I'm noticing, why the different prefixes `WP_Job_Manager` vs `WPJM` we should decide on a standard here and stick with it.